### ISSUE TITLE
Expose metrics

### ISSUE THREAD

**Issue by alexott** [datastax/dynamo-cassandra-proxy#2]
It would be nice to expose metrics, like:

- Number of requests
- same per table
- same per request type
- same per request type and table
- ...

**Comment by phact** [comment]
Good idea. Along the same vein, the java driver does expose some stats already that might be useful.
https://docs.datastax.com/en/developer/java-driver/4.2/manual/core/metrics/
